lib/Minz/Request.php

```diff
@@ -327,6 +332,10 @@ public static function getNotification() {
 	 *                > sinon, le dispatcher recharge en interne
 	 */
 	public static function forward($url = array(), $redirect = false) {
+		if (Minz_Request::originalRequest() === null && strpos('auth', json_encode($url)) !== false) {
```

Comment:
Cant' the `'auth'` be tested on the right $url array element without `json_encode()` ?

Reply:
It could if we were sure its an array but as per the line 339, we are not.

Reply:
The `strpos()` parameters were reversed. I believe this worked just by chance :-P
https://github.com/FreshRSS/FreshRSS/pull/5271